projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/vdk.py

```diff
@@ -68,3 +70,28 @@ def download_job(name: str, team: str, rest_api_url: str, path: str):
         cmd = JobDownloadSource(rest_api_url)
         cmd.download(team, name, path)
         return f"Downloaded the job with name {name} to {path}. "
+
+    # TODO: make it work with notebook jobs
+    @staticmethod
+    def create_job(
+        name: str, team: str, rest_api_url: str, path: str, local: bool, cloud: bool
+    ):
+        """
+        Execute `download job`.
+        :param name: the name of the data job that will be created
```

Comment:
Is the expectation like `vdk create` or `vdk download-job`?

Reply:
Oh, sorry I copy pasted the comment from the download job method and edited it but I missed this one. I will change it to "create job"